pkg/nimble/autoadv/nimble_autoadv.c

```diff
@@ -160,8 +160,14 @@ void nimble_autoadv_reset(void)
     int rc = 0;
     (void) rc;
 
-    rc = bluetil_ad_init_with_flags(&_ad, buf, sizeof(buf), BLUETIL_AD_FLAGS_DEFAULT);
-    assert(rc == BLUETIL_AD_OK);
+    if (IS_ACTIVE(NIMBLE_AUTOADV_FLAG_FIELD)) {
```

Comment:
Maybe even
```suggestion
    if (IS_ACTIVE(NIMBLE_AUTOADV_FLAG_FIELD) && BLUETIL_AD_FLAGS_DEFAULT) {
```
?

Reply:
Hmm this is always defined though since it's not actually a configuration(yet), so I guess it does not hurt adding this.

Reply:
Addressed squashed right away.